### ISSUE TITLE
Node v20.2.0 create error

### ISSUE THREAD

**Issue by sergical** [0xOlias/ponder#238]
```
pnpm create ponder
../Library/pnpm/store/v3/tmp/dlx-13731   |  +39 ++++
Packages are hard linked from the content-addressable store to the virtual store.
  Content-addressable store is at: /Users/sergiydybskiy/Library/pnpm/store/v3
  Virtual store is at:             ../Library/pnpm/store/v3/tmp/dlx-13731/node_modules/.pnpm
../Library/pnpm/store/v3/tmp/dlx-13731   | Progress: resolved 39, reused 39, downloaded 0, added 39, done
✔ What is your project named? … bera-graph-test
✔ Would you like to use a template for this project? › None

Creating a new Ponder app in /Users/sergiydybskiy/src/bera-graph-test.

Installing with pnpm.
node:internal/errors:862
  const err = new Error(message);
              ^

Error: Command failed: pnpm --silent install
    at checkExecSyncError (node:child_process:885:11)
    at execSync (node:child_process:957:15)
    at run (/Users/sergiydybskiy/Library/pnpm/store/v3/tmp/dlx-13731/node_modules/.pnpm/create-ponder@0.0.65/node_modules/create-ponder/dist/create-ponder.js:786:42)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async createPonder (/Users/sergiydybskiy/Library/pnpm/store/v3/tmp/dlx-13731/node_modules/.pnpm/create-ponder@0.0.65/node_modules/create-ponder/dist/create-ponder.js:928:3) {
  status: 1,
  signal: null,
  output: [ null, null, null ],
  pid: 13745,
  stdout: null,
  stderr: null
}

Node.js v20.2.0
 ERROR  Command failed with exit code 1: create-ponder

pnpm: Command failed with exit code 1: create-ponder
    at makeError (/Users/sergiydybskiy/Library/pnpm/global/5/.pnpm/pnpm@8.6.3/node_modules/pnpm/dist/pnpm.cjs:24241:17)
    at handlePromise (/Users/sergiydybskiy/Library/pnpm/global/5/.pnpm/pnpm@8.6.3/node_modules/pnpm/dist/pnpm.cjs:24812:33)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async Object.handler (/Users/sergiydybskiy/Library/pnpm/global/5/.pnpm/pnpm@8.6.3/node_modules/pnpm/dist/pnpm.cjs:207250:7)
    at async /Users/sergiydybskiy/Library/pnpm/global/5/.pnpm/pnpm@8.6.3/node_modules/pnpm/dist/pnpm.cjs:216656:21
    at async main (/Users/sergiydybskiy/Library/pnpm/global/5/.pnpm/pnpm@8.6.3/node_modules/pnpm/dist/pnpm.cjs:216623:34)
    at async runPnpm (/Users/sergiydybskiy/Library/pnpm/global/5/.pnpm/pnpm@8.6.3/node_modules/pnpm/dist/pnpm.cjs:216878:5)
    at async /Users/sergiydybskiy/Library/pnpm/global/5/.pnpm/pnpm@8.6.3/node_modules/pnpm/dist/pnpm.cjs:216870:7
```

**Comment by 0xOlias** [comment]
Thanks for reporting! I was able to reproduce this locally by running `pnpm create ponder` while using Node 20 (`nvm use 20`).

The problem is related to installing the dependency `better-sqlite3`. This library is unfortunately riddled with installation issues, I've peronsally spent hours on this. I opened a new issue here https://github.com/WiseLibs/better-sqlite3/issues/1027, hoping to get some clarity there so I can mitigate.
